src/quacks/__init__.py

```diff
@@ -80,14 +78,7 @@ def _is_classvar(t: type) -> bool:
         ...
 
 else:  # pragma: no cover
-    if sys.version_info < (3, 7):
-        from typing import _ClassVar
-
-        def _is_classvar(t: type) -> bool:
-            return type(t) is _ClassVar
+    from typing import ClassVar, _GenericAlias
```

Comment:
we can remove the TYPE_CHECKING case as well and move the imports to the top, now that there is no separate 3.6 case anymore

Reply:
also the coverage exemptions can be removed

Reply:
I tried, that but `mypy` complains about `_GenericAlias` not being present in `typing.pyi`. I assumed that why it was wrapped in the `TYPE_CHECKING` case.

Reply:
hmmm 🤔. Does adding `# type: ignore` after that particular import fix it?

Reply:
hmmm let me see if I can figure this out...

Reply:
on second thought: let's merge and fix separately